src/main/java/com/easysplit/ess/user/domain/models/User.java

```diff
@@ -13,14 +13,18 @@ public class User {
     private String name;
     private String lastname;
     private String username;
+    private String email;
+    private String phone;
     private Timestamp createdDate;
     private List<Link> links;
 
-    public User(String id, String name, String lastname, String username, Timestamp createdDate) {
+    public User(String id, String name, String lastname, String username, String email, String phone, Timestamp createdDate) {
         this.id = id;
         this.name = name;
         this.lastname = lastname;
         this.username = username;
+        this.email=email;
```

Comment:
Casi todas las asignaciones a variables tienen espacios que los separan `this.email = email`, lo dejamos asi o separados, si es juntos hay que cambiar todas